src/wp-admin/about.php

```diff
@@ -39,6 +39,37 @@
 			</nav>
 		</div>
 
+		<div class="about__section changelog">
+			<div class="column">
+				<h2><?php _e( 'Maintenance Release' ); ?></h2>
+				<p>
+					<?php
+					printf(
+						/* translators: 1: WordPress version number, 2: plural number of bugs. */
+						_n(
+							'<strong>Version %1$s</strong> addressed %2$s bug.',
+							'<strong>Version %1$s</strong> addressed %2$s bugs.',
+							666
+						),
+						'5.9.1',
+						number_format_i18n( 666 )
+					);
+					?>
```

Comment:
@audrasjb Can you please remove these PHP tags?

Reply:
While I'd theoretically agree with you, this code comes from previous versions and I assume it's probably better to keep a minimal consistency here…

- 5.8 https://github.com/WordPress/wordpress-develop/blob/5.8/src/wp-admin/about.php
- 5.7 https://github.com/WordPress/wordpress-develop/blob/5.7/src/wp-admin/about.php
- 5.6 https://github.com/WordPress/wordpress-develop/blob/5.8/src/wp-admin/about.php
- 5.5 https://github.com/WordPress/wordpress-develop/blob/5.8/src/wp-admin/about.php